### DISCUSSION TITLE
pymobiledevice3 remote tunneld - For How many devices it can create tunnel?

### DISCUSSION THREAD

**Discussion by mega6453** [doronz88/pymobiledevice3#1043] [Q&A] [answered]
When I run this command, with maximum of how many devices this command will work? For each device, do we need to run this separately or running just once enough and it will create tunnel for all the devices?
I don't have multiple devices right now, So I'm unable to test this so asking here..
Thanks.

**Comment by doronz88** [comment, marked as answer]
It doesn't have a limit on the connected devices. It's whole purpose is to be a RemoteXPC device manager.

**Reply by mega6453** [reply]
ok thanks..
